src/interpolation/Interpolant/LinearLogarithmic/test/interpolate.test.cpp

```diff
@@ -52,7 +52,9 @@ SCENARIO("The LinearLogarithmic interpolant computes is compatible with units",
   std::vector< double > xValues{ 1E-2, 1, 10 };
   auto f1 = []( auto x ){ return 13.2 * std::log(3.2 * x.value) * barn; };
   auto f2 = []( auto x ){ return -5.6 * std::log(2.4 * x.value) * barn; };
-
+  auto F1 = []( auto x ){return 13.2*x.value*(std::log(3.2*x.value)-1.) * barn * electronVolts;};
```

Comment:
You create `F1` and `F2` here, but do you use them anywhere?

Reply:
No, those should be removed from there. Initially, I had put the integration tests in the `interpolate.test.cpp` file, but later moved them to the `integration.test.cpp` file to keep things somewhat separated. I must have forgotten to remove these two lines. I will add a commit to remove them.